source/agora/consensus/EnrollmentManager.d

```diff
@@ -1264,28 +1266,29 @@ unittest
     assert(enrolls.length == 1);
 
     // make the enrollment a validator
-    man.addValidator(enroll, key_pair.address, Height(11), &utxo_set.peekUTXO, utxos);
+    man.addValidator(enroll, key_pair.address, Height(10), &utxo_set.peekUTXO, utxos);
     man.getEnrollments(enrolls, Height(11));
     assert(enrolls.length == 0);
 
     // add the enrollment that is already a validator, and check if
     // the enrollment can be nominated at the height before the cycle end
+    enroll = man.createEnrollment(utxo_set.keys[0], Height(10 + validator_cycle));
     assert(man.addEnrollment(enroll, key_pair.address, Height(11), &utxo_set.peekUTXO));
-    man.getEnrollments(enrolls, Height(validator_cycle + 9));
+    man.getEnrollments(enrolls, Height(validator_cycle + 8));
```

Comment:
I think it should be restored as it was because the change does not contain the original purpose.

Reply:
Ditto